statefun_tasks/_pipeline.py

```diff
@@ -230,7 +230,10 @@ def attempt_retry(self, context: _TaskContext, task_exception: TaskException):
 
         request = task_exception.original_request
         destination = context.get_caller_address()
-        context.pack_and_send_after(delay, destination, task_id, request)
+        if delay:
+            context.pack_and_send_after(delay, destination, task_id, request)
```

Comment:
This method fails if the delay is None or a zero timespan

Reply:
Ok